docker-compose.yml

```diff
@@ -553,10 +553,6 @@ services:
     - intranet-server
     - internet-server
 
-    # Connect to the custom default network
-    networks:
-    - network
-
```

Comment:
Please remove the earlier occurrence and keep this one to preserve the order that the `networks` key appears after the `depends_on` key.

Reply:
done 👍